packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -132,4 +132,71 @@ The `Button.Group` can be displayed vertically using the `vertical` prop.
 `Button.Group` can also be used to create a split button. <br />
 The dropdown can change the button action (as in this example) or be used to immediately trigger a related action.
 
-<Story id="inputs-button--group-split" />
\ No newline at end of file
+<Story id="inputs-button--group-split" />
+
+## Button.Toggle
+Is used to group related **options**.
+
+### Usage
+
+The `Button.Toggle`
+<ul>
+  <li>emphasizes groups of related <code>Toggle</code> buttons,</li>
+  <li>controls the selected state of its child buttons when given its own <code>selected</code> prop.</li>
+</ul>
+
+```tsx
+import { Button } from '@equinor/eds-core-react'
+import { save } from '@equinor/eds-icons'
+
+const [selectedButton, setSelectedButton] = useState([])
+const handleChange = (indexes: number[]) => {
+  setSelectedButton(indexes)
+}
+
```

Comment:
Suggest removing the "selected" and "onChange" in this example as they are not needed for this component to work. They are meant for more advance features if our users need it.